chainerx_cc/chainerx/python/routines.cc

```diff
@@ -608,6 +608,7 @@ void InitChainerxMath(pybind11::module& m) {
     m.def("tanh", [](const ArrayBodyPtr& x) { return MoveArrayBody(Tanh(Array{x})); }, py::arg("x"));
     m.def("sin", [](const ArrayBodyPtr& x) { return MoveArrayBody(Sin(Array{x})); }, py::arg("x"));
     m.def("cos", [](const ArrayBodyPtr& x) { return MoveArrayBody(Cos(Array{x})); }, py::arg("x"));
+    m.def("abs", [](const ArrayBodyPtr& x) { return MoveArrayBody(Absolute(Array{x})); }, py::arg("x"));
```

Comment:
I would prefer to align with Chainer in this case (and your C++ implementation), since for NumPy `np.abs is np.absolute`. What do you say?
```suggestion
    m.def("absolute", [](const ArrayBodyPtr& x) { return MoveArrayBody(Absolute(Array{x})); }, py::arg("x"));
```

Reply:
It might be fine also to alias. `m.attr("abs") = m.attr("absolute");`.